### DISCUSSION TITLE
No support for checkout using Service Principal

### DISCUSSION THREAD

**Discussion by lindamthomas** [community/community#45206] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

It is advised to use Service Principal (Functional Account) instead of the user's Personal Access Token (PAT) while automating CD pipelines. However, code checkout using Service Principal is not yet supported by GitHub. Would it be supported in near future?

**Comment by mikedrexler** [comment]
AT&T requested this very thing today.
